syncthing.te

```diff
@@ -0,0 +1,45 @@
+policy_module(syncthing, 1.0.0)
+
+# define types
+type syncthing_t;
+type syncthing_exec_t;
+userdom_user_application_domain(syncthing_t, syncthing_exec_t)
+
+type syncthing_config_home_t;
+userdom_user_home_content(syncthing_config_home_t)
+
+# file permissions
+fs_getattr_xattr_fs(syncthing_t)
+
+userdom_manage_user_home_content(syncthing_t)
+
+# read randomness
+dev_read_rand(syncthing_t)
+dev_read_urand(syncthing_t)
+
+# networking
+userdom_basic_networking(syncthing_t)
+
```

Comment:
On 08/21/2016 11:09 PM, Naftuli Tzvi Kay wrote:
>> +
>> +type syncthing_config_home_t;
>> +userdom_user_home_content(syncthing_config_home_t)
>> +
>> +# file permissions
>> +fs_getattr_xattr_fs(syncthing_t)
>> +
>> +userdom_manage_user_home_content(syncthing_t)
>> +
>> +# read randomness
>> +dev_read_rand(syncthing_t)
>> +dev_read_urand(syncthing_t)
>> +
>> +# networking
>> +userdom_basic_networking(syncthing_t)
>> +
> 
> `userdom_basic_networking`? Might be downstream in Fedora/RedHat land :disappointed: 
> 
> Can you suggest an alternate?
> 

Would need more information about what exactly prompted you to use this
(specific avc denials would be best)

-- 
Key fingerprint = 5F4D 3CDB D3F8 3652 FBD8  02D5 3B6C 5F1D 2C7B 6B02
https://sks-keyservers.net/pks/lookup?op=get&search=0x3B6C5F1D2C7B6B02
Dominick Grift